src/orquestra/sdk/_base/_storage/_basemodel.py

```diff
@@ -0,0 +1,29 @@
+################################################################################
+# © Copyright 2024 Zapata Computing Inc.
+################################################################################
+
+from typing import Any
+
+from pydantic.main import BaseModel
+
+
+class OrquestraBaseModel(BaseModel):
```

Comment:
That's a really good question. I believe it would impact anything that we both pickle and write to a file (Results, WorkflowDefs, etc.) and anything that might be nested within them (TaskDefs for example). I started reasoning through this, but decided to go with a blanket change because:
1. find-replace is fast and easy
2. I explicitly intend this hack to be as short lived as possible, c.f. [ORQSDK-2025]
3. If we have a mix of pickle-safe OrquestraBaseModel and unsafe BaseModel, it becomes easier to accidentally add something unpicklable later. The error message this causes is _not_ particularly clear, and we run the risk of wasting time solving these issues.